.all-contributorsrc

```diff
@@ -255,5 +256,6 @@
     }
   ],
   "contributorsPerLine": 7,
-  "commitConvention": "angular"
+  "commitConvention": "angular",
+  "commitType": "chore(all-contributors)"
```

Comment:
Updated this from default of `docs`. 

This is a new/undocumented property. Opened an issue with all-contributors to get some clarity on how to use it.